packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -175,7 +175,7 @@ abstract class ProjectBase extends Resource implements IProject {
    * May be unset, in which case this Project is not configured to use a VPC.
    * @internal
    */
-  protected _connections: ec2.Connections;
+  protected _connections: ec2.Connections | undefined;
```

Comment:
Why not `_connections?: ec2.Connections`?

Reply:
No particular reason. I like the explicitness of this.

Reply:
I prefer the `?` operator, only because it's consistent with everywhere else.